draft-song-dns-wireformat-http-01.txt

```diff
@@ -271,17 +274,17 @@ Internet-Draft          DNS wire-format over HTTP          February 2016
 
    Proxy-DNS-Transport: xyz
 
-   Where xyz is either UDP or TCP, which is the client's indication of
-   how it received the underlying DNS query, and which the server will
-   use when sending the query to the far-end DNS server.  This means if
 
 
 
-Song, et al.            Expires September 1, 2016               [Page 5]
+Song, et al.           Expires September 11, 2016               [Page 5]
 
-Internet-Draft          DNS wire-format over HTTP          February 2016
+Internet-Draft          DNS wire-format over HTTP             March 2016
 
 
+   Where xyz is either UDP or TCP, which is the client's indication of
+   how it received the underlying DNS query, and which the server will
```

Comment:
I don't think it's necessary. Every time "will" is used does not have to be WILL. That is mostly to let implementors know that if there is some chance when something doesn't seem like it necessarily has to be done, that it will be done. :wink: